rusk-wallet/src/lib/clients.rs

```diff
@@ -314,6 +314,7 @@ impl StateClient for State {
         })?
         .into_inner();
 
-        Ok((res.stake, res.expiration))
+        // TODO return a proper value here
+        Ok((res.value, 0))
```

Comment:
I assume we'll completely ditch the second parameter once `wallet-core` is updated?

Reply:
Once `wallet-core` is updated, the entire `Stake` structure (or equivalent) should be passed